src/containers/podcast/Podcast.js

```diff
@@ -3,10 +3,21 @@ import "./Podcast.css";
 import { podcastSection } from "../../portfolio";
 import { Fade } from "react-reveal";
 
+const cantDisplayError = "<div class='centerContent'><h2>Can't load? Check privacy protection settings</h2></div>";
+
+function timeOut(){
```

Comment:
The Twitter message implementation uses [Suspense](https://reactjs.org/docs/concurrent-mode-suspense.html) so you should try to do something similar here too with a fallback loader.

Reply:
Actually speaking it uses lazy loading, https://reactjs.org/docs/react-api.html#reactlazy, which in turn uses suspense.  See #181